demos/projects/ESPRESSIF/az-ca-recovery/config/demo_config.h

```diff
@@ -83,6 +83,23 @@
  */
 #define democonfigREGISTRATION_ID CONFIG_AZURE_DPS_REGISTRATION_ID
 
+/**
```

Comment:
Should this file get added to the style check as well?

Reply:
I think we should, but it seems we're inconsistent with adding the `config/` directory to it. I'll add a quick GH issue to do it separately in the future.

https://github.com/Azure-Samples/iot-middleware-freertos-samples/issues/316